tests/integrations/test_dsm.py

```diff
@@ -19,14 +20,19 @@ def setup_dsm_kafka(self):
     def test_dsm_kafka(self):
         assert self.r.text == "ok"
 
+        if context.library == "nodejs":
```

Comment:
Could you just add a comment explaining why it's different on node ? (and probably how those number are obtained? it would at least help me :) )

Reply:
just added a comment (: